doc/api/errors.md

```diff
@@ -599,182 +599,180 @@ found [here][online].
+The type of an asynchronous resource is invalid. Note that users are also able
+to define their own types when using the public embedder API.
 
 <a id="ERR_BUFFER_OUT_OF_BOUNDS"></a>
 ### ERR_BUFFER_OUT_OF_BOUNDS
 
-Used when attempting to perform an operation outside the bounds of a `Buffer`.
+An operation outside the bounds of a `Buffer` was attempted.
 
 <a id="ERR_BUFFER_TOO_LARGE"></a>
 ### ERR_BUFFER_TOO_LARGE
 
-Used when an attempt has been made to create a `Buffer` larger than the
-maximum allowed size.
+An attempt has been made to create a `Buffer` larger than the maximum allowed
+size.
 
 <a id="ERR_CANNOT_WATCH_SIGINT"></a>
 ### ERR_CANNOT_WATCH_SIGINT
 
-Used when Node.js is unable to watch for the `SIGINT` signal.
+Node.js was unable to watch for the `SIGINT` signal.
 
 <a id="ERR_CHILD_CLOSED_BEFORE_REPLY"></a>
 ### ERR_CHILD_CLOSED_BEFORE_REPLY
 
-Used when a child process is closed before the parent received a reply.
+A child process was closed before the parent received a reply.
 
 <a id="ERR_CONSOLE_WRITABLE_STREAM"></a>
 ### ERR_CONSOLE_WRITABLE_STREAM
 
-Used when `Console` is instantiated without `stdout` stream or when `stdout` or
-`stderr` streams are not writable.
+`Console` was instantiated without `stdout` stream, or `Console` has a
+non-writable `stdout` or `stderr` stream.
 
 <a id="ERR_CPU_USAGE"></a>
 ### ERR_CPU_USAGE
 
-Used when the native call from `process.cpuUsage` cannot be processed properly.
+The native call from `process.cpuUsage` could not be processed.
 
 <a id="ERR_CRYPTO_CUSTOM_ENGINE_NOT_SUPPORTED"></a>
 ### ERR_CRYPTO_CUSTOM_ENGINE_NOT_SUPPORTED
 
-Used when a client certificate engine is requested that is not supported by the
-version of OpenSSL being used.
+A client certificate engine was requested that is not supported by the version
+of OpenSSL being used.
 
 <a id="ERR_CRYPTO_ECDH_INVALID_FORMAT"></a>
 ### ERR_CRYPTO_ECDH_INVALID_FORMAT
 
-Used when an invalid value for the `format` argument has been passed to the
-`crypto.ECDH()` class `getPublicKey()` method.
+An invalid value for the `format` argument was passed to the `crypto.ECDH()`
+class `getPublicKey()` method.
 
 <a id="ERR_CRYPTO_ENGINE_UNKNOWN"></a>
 ### ERR_CRYPTO_ENGINE_UNKNOWN
 
-Used when an invalid crypto engine identifier is passed to
+An invalid crypto engine identifier was passed to
 [`require('crypto').setEngine()`][].
 
 <a id="ERR_CRYPTO_FIPS_FORCED"></a>
 ### ERR_CRYPTO_FIPS_FORCED
 
-Used when trying to enable or disable FIPS mode in the crypto module and
-the [`--force-fips`][] command-line argument is used.
+The [`--force-fips`][] command-line argument was used but there was an attempt
+to enable or disable FIPS mode in the `crypto` module.
 
 <a id="ERR_CRYPTO_FIPS_UNAVAILABLE"></a>
 ### ERR_CRYPTO_FIPS_UNAVAILABLE
 
-Used when trying to enable or disable FIPS mode when FIPS is not available.
+There was an attempt to enable or disable FIPS mode, but FIPS mode is not
```

Comment:
For consistency, `An attempt was made to enable...but FIPS mode was not available.`